benchmarks/java/src/main/java/com/google/protobuf/ProtoBench.java

```diff
@@ -97,16 +106,39 @@ public static boolean runTest(String file) {
     
     try {
       String args[] = new String[argsList.size()];
-      argsList.toArray(args);
-      CaliperMain.exitlessMain(args, 
+      argsList.toArray(args);  
+      CaliperMain.exitlessMain(args,
           new PrintWriter(System.out, true), new PrintWriter(System.err, true));
-      return true;
     } catch (Exception e) {
       System.err.println("Error: " + e.getMessage());
       System.err.println("Detailed exception information:");
       e.printStackTrace(System.err);
       return false;
     }
+    try {
+      double mininumScale = 0;
+      // If the file not exist, this will throw IOException, which won't print the warning 
+      // information below.
+      Scanner scanner = new Scanner(new String(readAllBytes("JavaBenchmarkWarning.txt")));
+      while (scanner.hasNext()) {
+        mininumScale = Math.max(mininumScale, scanner.nextDouble());
+      }
+      scanner.close();
+      
+      System.out.println(
+         "WARNING: This benchmark's whole iterations are not enough, consider to config caliper to "
+        + "run for more time to make the result more convincing. You may change the configure "
+        + "code in com.google.protobuf.ProtoBench.getCaliperOption() of benchmark " 
+        + benchmarkDataset.getMessageName()
+        + " to run for more time. e.g. Change the value of instrument.runtime.options.timingInterva"
+        + "l or value of instrument.runtime.options.measurements to be at least "
+        + Math.round(mininumScale * 10 + 1) / 10.0 
+        + " times of before, then build and run the benchmark again\n");
+      Files.deleteIfExists(Paths.get("JavaBenchmarkWarning.txt"));
+    } catch (IOException e) {
+      
```

Comment:
Add a comment here to explain why the exception is ignored.

Reply:
Done